tdrs-frontend/nginx/cloud.gov/locations.conf

```diff
@@ -22,6 +22,44 @@ location ~ ^/(v1|admin|static/admin|swagger|redocs) {
         add_header Access-Control-Allow-Origin 's3-us-gov-west-1.amazonaws.com';
 }
 
+location /kibana/ {
+    auth_request /kibana_auth_check;
+    auth_request_set $auth_status $upstream_status;
+
+    proxy_pass http://{{env "KIBANA_BASE_URL"}}:5601/;
+    proxy_pass_header x-csrftoken;
+    proxy_set_header   Host $host;
+    proxy_set_header   X-Real-IP         $remote_addr;
+    proxy_set_header   X-Forwarded-For   $proxy_add_x_forwarded_for;
+    proxy_set_header   X-Forwarded-Proto https;
+
+    proxy_connect_timeout 900;
+    proxy_read_timeout    300;
+    proxy_send_timeout    300;
+    send_timeout 	      900;
+    proxy_buffer_size     4k;
+    proxy_temp_file_write_size 64k;
+
+    proxy_hide_header Referrer-Policy;
+}
+
+location = /kibana_auth_check {
+    internal;
+    proxy_pass              http://{{env "BACKEND_HOST"}}.apps.internal:8080/kibana_auth_check/;
+    proxy_pass_header x-csrftoken;
+    proxy_set_header   Host $host;
+    proxy_set_header   X-Real-IP         $remote_addr;
+    proxy_set_header   X-Forwarded-For   $proxy_add_x_forwarded_for;
+    proxy_set_header   X-Forwarded-Proto https;
+
+    proxy_connect_timeout 900;
+    proxy_read_timeout    300;
+    proxy_send_timeout    300;
+    send_timeout 	      900;
+    proxy_buffer_size     4k;
```

Comment:
I don't think we need buffer for auth, also not the " proxy_temp_file_write_size"